synapse/storage/databases/main/keys.py

```diff
@@ -191,7 +194,9 @@ async def get_server_keys_json(
             A mapping from (server_name, key_id, source) triplets to a list of dicts
         """
 
-        def _get_server_keys_json_txn(txn):
+        def _get_server_keys_json_txn(
+            txn: LoggingTransaction,
+        ) -> Dict[Tuple[str, Optional[str], Optional[str]], List[dict]]:
```

Comment:
The last `dict` could be `Dict[str, Any]`: the keys are all column names. But given the return type of the outer function, this is probably fine as it is.

Reply:
I have changed this to `Dict[str, Any]`.